CSE-Hub/authentication/auth.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+require_once "./../sql_login/login.php";
+$conn = mysqli_connect($hostname,$username,$password,$database);
+
+if (!$conn) {
+    die('<p>Connection failed: <p>' . mysqli_connect_error());
+}
+if (!empty($_POST['name']) && !empty($_POST['pass'])) {
+    $unsafename = $_POST['name'];
+    $unsafepass = $_POST['pass'];
+
+    $query = $conn->prepare("SELECT * FROM {$table} WHERE {$userIds}=? AND {$userPasswords}=?");
+   
+    if ($query) {
+        $query->bind_param('ss', $unsafename, $unsafepass);

```

Comment:
- [ ]  Need a few comments ( at least one reference for sql injection in code ).